libs/user-facing-errors/src/query_engine.rs

```diff
@@ -298,3 +298,10 @@ pub struct InteractiveTransactionError {
 pub struct QueryParameterLimitExceeded {
     pub message: String,
 }
+
+#[derive(Debug, UserFacingError, Serialize)]
+#[user_facing(
+    code = "P2030",
+    message = "Cannot find a FULLTEXT index to use for the search, try adding a @@fulltext([Columns...]) to your schema"
```

Comment:
No need to use uppercase here, uppercase is very SQL-ly. Edit: Also true for all other occurrences of FULLTEXT down below.

Reply:
`columns` is also SQL specific, `fields` is the generic Prisma way.